apps/web/components/sample-test/Button.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from "@testing-library/react";
+import { describe, test, expect } from "vitest";
+import "@testing-library/jest-dom";
```

Comment:
With your setup, each file should contain this line to support jest-dom assertions. You can optimize by adding it to the setup file

Reply:
Add this to your `vitest.config.mts`

`setupFiles: ["./vitest-setup.ts"],`

Reply:
`vitest-setup.ts`

```ts
import "@testing-library/jest-dom/vitest";
```